js/script.js

```diff
@@ -1,9 +1,33 @@
 import { $createButton, $taskContainer } from "./element.js";
 import { sanitizeInput, getUniqueId } from "./utlity.js";
+import { READ, EDIT } from "./const.js";
 
 let isFormOpen = false;
 let tasks = [];
 
+const editForm = (liElement, task) => {
+  const $inputElement = document.createElement("input");
+  const $updateButton = document.createElement("button");
+  const $cancelButton = document.createElement("button");
+  const $deleteButton = document.createElement("button");
+
+  $inputElement.setAttribute("id", `input-${task.id}`);
+
+  $inputElement.value = task.title;
+
+  $deleteButton.innerText = "Delete";
+  $updateButton.innerHTML = "Update";
+  $cancelButton.innerHTML = "Cancel";
+
+  $deleteButton.addEventListener("click", () => deleteTaskHandler(task.id));
+  $updateButton.addEventListener("click", () => updateTaskHandler(task.id));
+  $cancelButton.addEventListener("click", () => calcelTaskHandler(task.id));
+
+  liElement.append($inputElement, $updateButton, $deleteButton, $cancelButton);
+
+  return liElement;
+};
+
```

Comment:
Bhai, I think, we are recreating same button elements again in this function, like in `createForm` function. Another thing to create button, input, or span or any other element should be sole responsibility for `createForm` function. we can pass the reference of those required elements as arguments to another function. May be we don't need to `return liElement;` statement, as currently we aren't using it. In fact, I think we can remove this function. Since, if we remove all the element creation there is only two lines of code which can be written in `else` block.